packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -1990,7 +2013,7 @@ function ScatterplotViz(props: VisualizationProps<Options>) {
     if (!dataElementConstraints) return false;
     return Object.entries(dataElementConstraints[0])
       .filter((variable) => variable[1].isRequired)
-      .every((reqdVar) => !!(vizConfig as any)[reqdVar[0]]);
+      .every((reqdVar) => !!vizConfig[reqdVar[0] as keyof typeof vizConfig]);
```

Comment:
I think you can use `ScatterplotConfig` here
```suggestion
      .every((reqdVar) => !!vizConfig[reqdVar[0] as keyof ScatterplotConfig]);
```

Reply:
👍 complied